src/utils/geo.js

```diff
@@ -0,0 +1,202 @@
+import Cookies from 'js-cookie';
+import Random from './random';
+
+const earth = 'XX',
+	negativePrefix = 'non-',
+	samplingChar = '/';
+
+let geoData = null,
+	cache = [];
+
+function isCountryWithSampling(geo) {
```

Comment:
It isn't really country, right? Can we rename it to `hasSampling`? Or something similar